index.html

```diff
@@ -143,14 +159,16 @@ <h2 class="presentation">Multi-Post Stories</h2>
             <li><button type="button" class="language">javascript</button></li>
         </ul>
         <br>
-        <fieldset class="project"><button type="button" class="button_project">See Project</button></fieldset>
+        <fieldset class="project"><button type="button" class="button_project">See Project</button></fieldset></div>
 </div></div>
 </section>
-<!-- work section finish-->
+<!-- middle section finish-->
 <!-- about myself start-->
-<section class="about_myself">
-    <h2 class="presentation2">About <br> myself <br></h2>
+<section class="about_myself" id="about">
 <div class="section1">
+   <div class="ams12">
+    <h2 class="presentation2">About <br> myself <br></h2>
+
```

Comment:
- [ ] Hey, Please also kindly make sure your about section dominates your body background here also. check the images to compare so you can fix the issue with your design compared to that of the Figma design.

:point_down:  **YOUR DESIGN**

![about](https://user-images.githubusercontent.com/59397159/165690092-127b6c11-8b65-43ae-b0ae-d396be1a0ab1.png)

:point_down: **FIGMA DESIGN**

![figAbout](https://user-images.githubusercontent.com/59397159/165690178-1ef10a18-81b6-405c-943e-22746717d17d.png)